### ISSUE TITLE
Constant errors when configuring risk management policies

### ISSUE THREAD

**Issue by Naascap** [melonproject/melon-lab#357]
![image](https://user-images.githubusercontent.com/56631331/66906527-2bfe4700-f008-11e9-972e-e0ddaad7986f.png)

**Comment by Naascap** [comment]
Same issue when trading

**Comment by jennazenk** [comment]
Can you give us some details on your OS and the version of MMI you're using?

**Comment by Naascap** [comment]
Windows 10, using MMI 1.03

**Comment by fubhy** [comment]
@Naascap Can you confirm that this same error occurs when trading and when deploying the risk management policies? The error message stems from a failed contract deployment which would be unrelated to anything that happens in the trading related transactions.

Does this affect all risk management policies for you?

**Comment by Naascap** [comment]
@fubhy Yes both trading and risk management policies have the same problem.

**Comment by fubhy** [comment]
Can you check if this is resolved with the temporary fix I described here: https://github.com/melonproject/melon-lab/issues/352#issuecomment-542664925

**Comment by fubhy** [comment]
@Naascap That error hints at a problem between Frame and your hardware wallet. It means that you've previously selected your account in Frame (so the connection to your wallet once was fine) but then when the transaction was going to be signed, it could not find your hardware wallet anymore. Either because the USB connection was interrupted, the device went to sleep or you left the Ethereum app on the device, etc. It usually can be resolved by unplugging the device once and unlocking it again.

**Comment by Naascap** [comment]
@fubhy Unfortunately I've attempted the solution above, but I'm still getting the same error. Could it be because I'm using a Ledger X?
